SubjuGator/gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -116,8 +142,14 @@ def get_ranges(self) -> Tuple[np.ndarray, np.ndarray]:
         )
         return minima, maxima
 
-    def get_thruster_wrench(self, position, direction):
-        """Compute a single column of B, or the wrench created by a particular thruster"""
+    def get_thruster_wrench(self, position: list, direction: list) -> np.ndarray:
+        """Compute a single column of B, or the wrench created by a particular thruster
```

Comment:
Descriptions should start on a new line for docstrings longer than 1 line. Could we move the sentence to the next line, leaving the triple quotes where they are?